Dockerfile

```diff
@@ -61,25 +61,30 @@ RUN cd munimap_transport && python setup.py clean && python setup.py egg_info sd
 
 
 
-FROM python:2.7.16-jessie as RUNNER
+FROM python:3.9.13-buster as RUNNER
 
 # TODO check which libs are actually needed
 RUN apt-get update && apt-get install -y \
     build-essential \
-    python-dev \
-    openjdk-7-jre-headless \
+    python3-dev \
+    libpython3-dev \
+    python-gdal \
+    python3-gdal \
```

Comment:
Do we need both `python-gdal` and `python3-gdal`?

Reply:
Strange, I was sure I deleted it. Probably did not commit this change. I Will get rid of it. Only python3 variant is needed